<launch>

  <env name="LIBGL_ALWAYS_SOFTWARE" value="1"/>

  <!-- read value of arg -->
  <param name="param" value="$(arg filter_config)"/> 
  
  <rosparam file="$(find seam_detection)/config/$(arg filter_config).yaml" />

  <!-- setup the transformations and frames for the system-->
  <node
    pkg="seam_detection"
    name="seam_detection_tf"
    type="seam_detection_tf"
    output="screen"
  />                      
  
  <node name="rviz" 
    pkg="rviz" 
    type="rviz" 
    args="-d $(find seam_detection)/rviz/filter_cloud.rviz">
  </node>

  <node                     
    pkg="seam_detection"          
    name="filter_cloud"   
    type="filter_cloud"
    output="screen"  
   />
   
</launch>
